packages/react-storage/src/components/StorageBrowser/validators/assertListCallerAccessGrantInput.ts

```diff
@@ -0,0 +1,9 @@
+export function assertListCallerAccessGrantInput(
+  value: string | undefined
+): asserts value is string {
+  if (!value) {
+    throw new Error(
+      'Storage Browser: Must provide accountId provided for `listCallerAccessGrants`.'
```

Comment:
This error message is a little confusing. Are we just saying they didn't provide an accountId?
```suggestion
      'Storage Browser: Must provide accountId to `listCallerAccessGrants`.'
```

Reply:
If you have to eat 🍕 everynight?